A50-xds-outlier-detection.md

```diff
@@ -126,10 +142,18 @@ The `outlier_detection` LB policy will provide a picker that delegates to the ch
 The `outlier_detection` LB policy will have a timer that triggers on a period determined by the `interval` config option, and does the following:
 
  1. For each address, swap the call counter's buckets in that address's map entry.
- 2. Run the [success rate](#success-rate-algorithm) and [falure percentage](#failure-percentage-algorithm) algorithms. To eject an address, it will set the address's ejection timestamp to the current time, increment the multiplier, and call `eject()` on each subchannel wrapper.
- 3. For each address in the map:
+ 2. If the `success_rate_ejection` configuration field is set, run the [success rate](#success-rate-algorithm) algorithm.
+ 3. If the `failure_percentage_ejection` configuration field is set, run the [falure percentage](#failure-percentage-algorithm) algorithm.
+ 4. Run the [success rate](#success-rate-algorithm) and [falure percentage](#failure-percentage-algorithm) algorithms.
```

Comment:
This should now be removed in favor of the previous two, right?

Reply:
Yes, I took it out now.